extensions/applicationinsights-debugplugin-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/applicationinsights-debugplugin-js",
-    "version": "2.5.4",
+    "version": "2.5.5-beta",
```

Comment:
Will actually update this to 2.5.6-beta as part of preparing and revving the version number in a follow up PR